tests/02_schemas/test_unit_schemas.py

```diff
@@ -141,6 +145,20 @@ def test_apply_workflow_read():
     assert job2.output_dataset_id is None
     assert job2.workflow_id is None
 
+    job3 = ApplyWorkflowRead(
+        id=1,
+        start_timestamp=datetime(2000, 1, 1, tzinfo=None),
+        end_timestamp=datetime(2000, 1, 2, tzinfo=None),
+        status="good",
+        project_dump=PROJECT_DUMP,
+        workflow_dump=WORKFLOW_DUMP,
+        input_dataset_dump=DATASET_DUMP,
+        output_dataset_dump=DATASET_DUMP,
+        user_email="test@fractal.com",
+    )
+    assert job3.start_timestamp.tzinfo == timezone.utc
```

Comment:
Could we add a comment here? Something like "Because of `valutc` validator, the actual attribute is timezone-aware".